doc/manuals/admin/perf_tuning.md

```diff
@@ -8,7 +8,8 @@
 * [Thread pool considerations](#thread-pool-considerations)
 * [Identifying bootlenecks looking at semWait statistics](#identifying-bootlenecks-looking-at-semwait-statistics)
 * [Log impact in performance](#log-impact-in-performance)
-* [Mutex policy impact in performance](#mutex-policy-impact-in-performance)
+* [Metrics impact in performance](#metrics-impact-in-performance)
```

Comment:
This typo I guess we have in 'master' as well ...
impact in performance => impact on performance